source/gems/dry-validation/basics/macros.html.md

```diff
@@ -9,7 +9,9 @@ This document describes available built-in macros.
 
 ### filled
 
-Use it when a value is expected to be filled.
+Use it when a value is expected to be filled. "filled" means that the value is
+non-nil, and if the value is a `String`, `Hash`, or `Array` then `value.empty?`
+returns `false`.
```

Comment:
fixed the wrapping, changed the wording to something I feel is a bit clearer, and rebased.

What about my comments above about `build`? Should I also be making (presumably automated) changes to `/docs` in these kind of PRs, or is it enough to just commit changes to `/source`?